### PR TITLE
Ensure HANA_INST_DEV is clean before configuration

### PR DESCRIPTION
**Pull request by alvarocarvajald**
It seems changes introduced in #11325 are causing HANA tests on IPMI with an specific device set on `HANA_INST_DEV` to fail as the partition still contains some LVM information from the previous test.

Current guess is that the `wipefs -a` calls introduced in #11325 for IPMI wipe enough information from the disk so the installer detects it as an empty disk, but leave LVM information intact at least in the third partition which make the `hana_install` module fail while creating a PV.

This PR introduces a `wipefs` call directly on the new partition (created in the same module) to ensure that it contains no LVM information, while also preemptively removing any trace of the VG and LVs that may have remained in the disks from a previous test.

- Failed test: https://openqa.suse.de/tests/4932797#step/hana_install/57
- Related ticket: N/A
- Needles: N/A
- Verification run: https://openqa.suse.de/t4942236 (15-SP3 Build 63.1 Alpha4, HANA2 SPS04rev46), https://openqa.suse.de/t4942269 (15-SP3 Build 72.1, HANA2 SPS04rev46), https://openqa.suse.de/t4942268 (15-SP3 Build 72.1, HANA2 SPS04rev48) - Failures in build 72.1 are due to bsc#1178366 and not related to this PR
- Regression tests: [x86_64 on qemu](https://openqa.suse.de/t4942233), HanaSR on Power9 ppc64le: http://mango.qa.suse.de/tests/3252 & http://mango.qa.suse.de/tests/3253

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]
LGTM

**Review by juadk — APPROVED** [review]
Nothing to say except LGTM! thx

File: tests/sles4sap/hana_install.pm [hunk 1]
```diff
@@ -127,6 +128,12 @@ sub run {
                 assert_script_run "parted --script $device --wipesignatures -- mklabel gpt mkpart primary 1 -1";
                 $device .= is_multipath() ? '-part1' : '1';
             }
+            # Remove traces of LVM structures from previous tests before configuring
+            script_run 'lvremove -f vg_hana';
```
Comment (ldevulder) [inline tests/sles4sap/hana_install.pm:132]: Maybe `vg_hana` could be added in a variable, in that case it will be easy to change his name if needed, WDYT?

**Review by ldevulder — APPROVED** [review]
LGTM! Thx! Just a small comment ;-)

File: tests/sles4sap/hana_install.pm (hunk 1 above)
Comment (alvarocarvajald) [reply tests/sles4sap/hana_install.pm]: Done. I added only a regression verification run in x86_64 on qemu for the changes: https://openqa.suse.de/tests/4944446

**Pull request merged**